Quizz-app/src/Components/Quiz/Quiz.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { useRef } from 'react'
+import { useState } from 'react'
+import './Quiz.css'
+import { data } from '../../assets/data/data'
+
+const Quiz = () => {
+
+    let [index, setIndex] = useState(0);
+    let [question, setQuestion] = useState(data[index]);
+    let [lock, setlock] = useState(false);
+    let [score, setScore] = useState(0);
+
+    let option1 = useRef(null);
+    let option2 = useRef(null);
+    let option3 = useRef(null);
+    let option4 = useRef(null);
+
+    let optionArray = [option1, option2, option3, option4];
+    
+    const checkAnswer = (e, answer) => {
+        if (lock === false) {
+            e.target.classList.add("lock")
+            if (question.answer === answer) {
+                e.target.classList.add("correct");
+                setlock(true);
+                setScore(score + 1);
+            } else {
+                e.target.classList.add("wrong");
+                setlock(true);
+                optionArray[question.answer].current.classList.add("correct");
+            }
+        }
+
+    }
+
+    const refreshQuiz = () => {
+        setIndex(0);
+        setQuestion(data[0]);
+        setlock(false);
+        setScore(0);
+        optionArray.forEach((option) => {
+            option.current.classList.remove("lock");
+            option.current.classList.remove("correct");
+            option.current.classList.remove("wrong");
+        })
+    }
+
+    const nextQuestion = () => {
+        if (index < data.length - 1) {
+            setIndex(index + 1);
+            setQuestion(data[index + 1]);
+            setlock(false);
+            optionArray.forEach((option) => {
+                option.current.classList.remove("lock");
+                option.current.classList.remove("correct");
+                option.current.classList.remove("wrong");
```

Comment:
Poți șterge mai multe clase în același call.

```suggestion
                option.current.classList.remove("lock", "correct", "wrong");
```